packages/core/contracts/data-verification-mechanism/implementation/Staker.sol

```diff
@@ -286,12 +286,12 @@ abstract contract Staker is StakerInterface, Ownable, Lockable, MultiCaller {
 
     /**
      * @notice Gets the pending stake for a voter for a given round.
-     * @param voterAddress the voter address.
+     * @param voter the voter address.
      * @param roundId round id.
      * @return uint256 amount of the pending stake.
      */
-    function getVoterPendingStake(address voterAddress, uint256 roundId) external view returns (uint256) {
```

Comment:
pendingStakes is a `mapping(uint256 => uint256) pendingStakes`. Why do he have a `uint32` here?

https://github.com/UMAprotocol/protocol/blob/d6bb02cb3d9b765d820c7547256c55bcdc1f9822/packages/core/contracts/data-verification-mechanism/implementation/Staker.sol#L26

Reply:
good catch. cant be `uint32 => uint256`